inference-contract/README.md

```diff
@@ -107,3 +107,40 @@ the inferencing operations.
 cd $PDO_CONTRACTS_SOURCE_ROOT/inference-contract/test/
 ./script_test.sh
 ```
+
+## Jupyter Notebooks for the Inference Contract ##
+
+We provide [Jupyter Notebooks](./docs/notebooks/README.md) that can be executed
+in an interactive manner to illustrate the functionality of the inference
+contract family. We support Jupyter notebook deployment via both docker containers
+as well as bare-metal. For docker, please invoke the `make test_jupyter` command
+from within the docker folder contained within this repo. This command deploys the
+pdo ledger, pdo services, guardian frontend as well as the openvino containers in
```

Comment:
Please ensure consistent capitalization here an in other docs.

```suggestion
pdo ledger, pdo services, guardian frontend as well as the openVINO containers in
```

Reply:
thanks, done